examples/PhasorDynamics/Example2/example2.cpp

```diff
@@ -0,0 +1,125 @@
+#include "example2.hpp"
+
+#include <cmath>
+#include <cstdio>
+#include <time.h>
+#include <vector>
+
+#include <Model/PhasorDynamics/Branch/Branch.hpp>
+#include <Model/PhasorDynamics/Bus/Bus.hpp>
+#include <Model/PhasorDynamics/Bus/BusInfinite.hpp>
+#include <Model/PhasorDynamics/BusFault/BusFault.hpp>
+#include <Model/PhasorDynamics/Load/Load.hpp>
+#include <Model/PhasorDynamics/SynchronousMachine/GENROUwS/Genrou.hpp>
+#include <Model/PhasorDynamics/SystemModel.hpp>
+#include <Solver/Dynamic/Ida.hpp>
+#include <Utilities/Testing.hpp>
+
+#define _CRT_SECURE_NO_WARNINGS
+#define _USE_MATH_DEFINES
+
+int main()
+{
+  using namespace GridKit::PhasorDynamics;
+  using namespace AnalysisManager::Sundials;
+  using scalar_type = double;
+  using real_type   = double;
+  using index_type  = size_t;
+
+  auto error_allowed = static_cast<real_type>(1e-4);
+
+  std::cout << "Example 2 version 1\n";
+
+  /* Create model parts */
+  SystemModel<scalar_type, index_type> sys;
+  BusInfinite<scalar_type, index_type> bus1(1.06, 0.0);
+  Bus<scalar_type, index_type>         bus2(1.0599558398065716, -0.009675621941024773);
+  Bus<scalar_type, index_type>         bus3(0.9610827543495831, -0.13122476630506485);
+  Branch<scalar_type, index_type>      branch12(&bus1, &bus2, 0.05, 0.21, 0, 0.1);
+  Branch<scalar_type, index_type>      branch13(&bus1, &bus3, 0.06, 0.15, 0, 0.12);
+  Branch<scalar_type, index_type>      branch23(&bus2, &bus3, 0.08, 0.27, 0, 0.45);
+  Genrou<scalar_type, index_type>      gen2(&bus2, 1, 0.5, -0.07588, 2.7, 0., 0., 7., .04, .05, .75, 1.9, 0.17, 0.15, 0.4, 0.35, 0.15, 0.14999, 0., 0.);
+  Genrou<scalar_type, index_type>      gen3(&bus3, 1, 0.25, 0.26587, 1.6, 0., 0., 7.5, .04, .05, .75, 2.3, 0.2, 0.18, 0.5, 0.5, 0.18, 0.15, 0., 0.);
+  Load<scalar_type, index_type>        load3(&bus3, 0.4447197839297772, 0.20330047265361242);
+  BusFault<scalar_type, index_type>    fault(&bus3, 0, 1e-5, 0);
+
+  /* Connect everything together */
+  sys.addBus(&bus1);
+  sys.addBus(&bus2);
+  sys.addBus(&bus3);
+  sys.addComponent(&branch12);
+  sys.addComponent(&branch13);
+  sys.addComponent(&branch23);
+  sys.addComponent(&fault);
+  sys.addComponent(&load3);
+  sys.addComponent(&gen2);
+  sys.addComponent(&gen3);
+  sys.allocate();
+
+  real_type dt = 1.0 / 4.0 / 60.0;
+
+  std::stringstream buffer;
+
+  /* Set up simulation */
+  Ida<scalar_type, index_type> ida(&sys);
+  ida.configureSimulation();
+
+  /* Run simulation */
+  scalar_type start = static_cast<scalar_type>(clock());
+  // ida.printOutputF(0, 0, buffer);
+  ida.initializeSimulation(0.0, false);
+  ida.runSimulationFixed(0.0, dt, 1.0, buffer);
```

Comment:
I really don't like this `buffer` solution. It would be better to put these values in a data structure.

Reply:
Let's include this in #91 scope.